vizier/backend/src/info/vizierdb/commands/plot/PlotUtils.scala

```diff
@@ -102,6 +137,15 @@ object PlotUtils
       }
     }
 
+    if (isBarChart) {
```

Comment:
A little bit of code re-organization here is warranted to preserve consistency.

- `sort` and `isBarChart` are implemented as parameters of `makeSeries`
- `aggregateSeries` is implemented as a class method of `Series`.

All three serve nearly identical roles: applying preprocessing edits to the dataframe.  All three should be implemented the same way.

Since we made `rows` a lazy variable of Series, both approaches should have comparable runtimes and overheads.  I'm partial to making them class methods of `Series` in the style of `aggregateSeries`, since this approach offers a bit more flexibility than putting everything into one mega-constructor.

Reply:
not yet resolved.